test/interchaintest/interchainstaking_test.go

```diff
@@ -102,32 +118,176 @@ func TestInterchainStaking(t *testing.T) {
 		},
 	)
 
+	// Get all Validators
+	stdout, _, err := gaia.Validators[0].ExecQuery(ctx, "staking", "validators")
+	require.NoError(t, err)
+	require.NotEmpty(t, stdout)
+	t.Logf("Validators: %s", string(stdout))
+
+	var validatorsResp QueryValidatorsResponse
+	err = codec.NewLegacyAmino().UnmarshalJSON(stdout, &validatorsResp)
+	require.NoError(t, err)
+
+	gaiaValidators := validatorsResp.Validators
+	fmt.Println(gaiaValidators)
```

Comment:
You should delete Print funcs after test in local. It can take a long time for test in CI